util/src/main/java/com/google/cloud/hadoop/util/RetryHttpInitializer.java

```diff
@@ -80,14 +80,17 @@ public void initialize(HttpRequest request) throws IOException {
       credentials.initialize(request);
     }
 
+    RequestTracker tracker = getRequestTracker(request);
```

Comment:
A Retired request will get initialized too and will initialize a separate tracker for it, correct?  

How we are able to maintain the retryCount in RequestTracker then?